config/core/test-configs.yaml

```diff
@@ -2257,6 +2257,18 @@ test_configs:
   - device_type: meson-gxl-s905x-libretech-cc
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-alsa
+      - kselftest-cpufreq
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ipc
+      # ltp-mm - Runs board out of memory
+      - ltp-pty
+      - ltp-timers
```

Comment:
OK with a couple of failing test cases https://staging.kernelci.org/test/plan/id/626135ed6dcd090e2605f4c4/